python/rmm/pyproject.toml

```diff
@@ -125,6 +121,18 @@ provider = "scikit_build_core.metadata.regex"
 input = "rmm/VERSION"
 regex = "(?P<value>.*)"
 
+[tool.rapids-build-backend]
+build-backend = "scikit_build_core.build"
+commit-file = "rmm/GIT_COMMIT"
+dependencies-file = "../../dependencies.yaml"
+requires = [
+    "cmake>=3.26.4",
+    "cuda-python>=11.7.1,<12.0a0",
```

Comment:
At some point I'd like to consider switching the "default" pyproject.toml to CUDA 12. That can be evaluated later but if we like the idea we could file a build-planning issue.